src/USER-CGDNA/fix_nve_dot.h

```diff
@@ -10,6 +10,9 @@
 
    See the README file in the top-level LAMMPS directory.
 ------------------------------------------------------------------------- */
+/* ----------------------------------------------------------------------
```

Comment:
OK, I'll remove these lines. It's just it was included in some of the other headers before.